src/nft_contract/src/main.nr

```diff
@@ -63,6 +63,12 @@ pub contract NFT {
      * ========================= PRIVATE =========================
      * ======================================================== */
 
+    /// @notice Transfers token by id from private nft set to a public owner
```

Comment:
i think the user should be abstracted from the "private set' concept, and also think that whoever reads this is a dev, but in the end in etherscan the user does get these `@notice` tags so it could propagate, let's use private owner and public owner